benchbuild/utils/__init__.py

```diff
@@ -5,8 +5,10 @@
 """
 import logging
 import sys
+import typing as tp
 from types import ModuleType
 
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)